cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -40,7 +40,7 @@ import (
 
 // Exponential backoff for MemberAdd/Remove (values exclude jitter):
 // 0, 50, 150, 350, 750, 1550, 3150, 6350, 12750 ms
-var addRemoveBackoff = wait.Backoff{
+var etcdBackoff = wait.Backoff{
 	Steps:    8,
 	Duration: 50 * time.Millisecond,
 	Factor:   2.0,
```

Comment:
we need to increase this, let me fetch the new values.

Reply:
done